net/mdns-repeater/src/etc/inc/plugins.inc.d/mdnsrepeater.inc

```diff
@@ -28,12 +28,21 @@
 
 function mdnsrepeater_enabled()
 {
-    $mdns_repeater = new \OPNsense\MDNSRepeater\MDNSRepeater();
-    if ((string)$mdns_repeater->enabled == '1') {
-        return true;
-    }
+    $model = new \OPNsense\MDNSRepeater\MDNSRepeater();
```

Comment:
Usually I don’t mind these refactors, but if you have a feature to add or bug to fix it makes review shorter and quicker

Reply:
I moved the small cosmetic changes to https://github.com/opnsense/plugins/pull/2763 for easier review.